docs/devguide/create-metricset.asciidoc

```diff
@@ -10,14 +10,16 @@ implementation of existing metricsets for inspiration.
 
 To create a new metricset:
 
-. Run the following command inside your beat directory:
+. Run the following command inside the metricbeat directory:
 +
 [source,bash]
 ----
 make create-metricset
 ----
 +
-You'll be prompted to enter a module and metricset name. Only use characters `[a-z]`
+You need to use Python 2.7 to run this command. If you also have Python 3 in your system, you can run the command with `VIRTUALENV_PARAMS="-p python2.7" make create-metricset` to force it to run with Python 2.7.
```

Comment:
Typically we use asciidoc attributes to resolve version numbers, or this info gets stale and we forget to update it. We should use one of the existing attributes, or add one to this file:  https://github.com/elastic/beats/blob/master/libbeat/docs/version.asciidoc

So you might add something like `:python-short: python2.7` to the versions file, and then use `Python {python-short}` to resolve the value at doc build time.

Note that asciidoc attributes that appear within code snippets need extra love to resolve correctly. Wrap the code in plus signs instead of back ticks. 

```
+VIRTUALENV_PARAMS="-p python{python-short}" make create-metricset+

Reply:
Ok, I found a `python` reference already which also works so I have changed it accordingly.